docs/development/test-vectors.rst

```diff
@@ -625,6 +625,108 @@ Custom PKCS12 Test Vectors
 * ``pkcs12/cert-aes256cbc-no-key.p12`` - A PKCS12 file containing a cert
   (``x509/custom/ca/ca.pem``) encrypted via AES 256 CBC with the
   password ``cryptography`` and no private key.
+* ``pkcs12/no-name-no-pwd.p12`` - A PKCS12 file containing a cert
+  (``x509/custom/ca/ca.pem``) and key (``x509/custom/ca/ca_key.pem``),
+  as well as two additional certificates (``x509/custom/dsa_selfsigned_ca.pem``
```

Comment:
Looks like the docs need updating still?

Reply:
:laughing: it's early in the morning for me... :)

Reply:
Fixed in cdc2cd7201df408427cd92d1eddb1329b9b56799.